# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: 907d8172c495
# Full Hash: 907d8172c495b97e4dcfaf641209ebd2552fc509
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-07-11 21:52:53
# Regressor Bug: 1557983
# File Overlap Count: 1
# Description:
#   Bug 1557983. Remove nsPrintJob::HasPrintCallbackCanvas.
#   
#   After refactoring the 'beforeprint'/'afterprint' code, this method is not used
#   any more.
#   
# ==============================================================================

diff -r 25fa445f9702 -r 907d8172c495 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sat Jul 11 14:14:03 2020 +0000
+++ b/layout/printing/nsPrintJob.h	Sat Jul 11 14:14:15 2020 +0000
@@ -131,12 +131,6 @@
 
   void TurnScriptingOn(bool aDoTurnOn);
 
-  /**
-   * Checks to see if the document this print engine is associated with has any
-   * canvases that have a mozPrintCallback.
-   * https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement#Properties
-   */
-  bool HasPrintCallbackCanvas() { return mHasMozPrintCallback; }
   bool PrePrintPage();
   bool PrintPage(nsPrintObject* aPOect, bool& aInRange);
   bool DonePrintingPages(nsPrintObject* aPO, nsresult aResult);
